app/cdap/components/WrangleHome/Components/OngoingDataExploration/utils.ts

```diff
@@ -0,0 +1,37 @@
+import { ImportDatasetIcon } from '../WrangleCard/iconStore/ImportDatasetIcon';
+
+export const updatedData = (olddata: any) => {
```

Comment:
Please add a more descriptive name or a comment to say what this is doing.

Reply:
added the descriptive name and comment